lib/LINE/Bot/API/Client/Furl.pm

```diff
@@ -159,6 +159,31 @@ sub post_image {
     $ret;
 }
 
+sub post_file {
+    my ($self, $url, $body, $file_path) = @_;
+
+    open my $fh, '<', $file_path
+        or croak 'Failed to open file.';
```

Comment:
It'd be nice to include  `$file_path` in the error message -- so when error happes, users can track it down.

Reply:
That's true... I will include it :)